src/rust/Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN --mount=type=cache,target=/var/lib/apt/lists,sharing=locked,id=rust-base-apt
 # - in plugin-registry image
 WORKDIR /nomad
 RUN <<EOF
-NOMAD_VERSION="1.2.4"
+NOMAD_VERSION="1.3.5"
```

Comment:
should this be `"1.3.5-1"` to match what's in etc/chromeos/lib/installs.sh?

Reply:
The `-1` at the end is a linux repository thing, and doesn't appear to be used in the releases page